Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -7999,9 +7990,10 @@ class BBQJIT {
         case SIMDLaneOperation::Add:
             m_jit.vectorAdd(info, leftLocation.asFPR(), rightLocation.asFPR(), resultLocation.asFPR());
             return { };
-        case SIMDLaneOperation::Mul:
+        case SIMDLaneOperation::Mul: {
```

Comment:
Why the added { } here but not anywhere else?  I don't think these are needed.

Reply:
Removed